docs/source-2.0/aws/protocols/aws-json-1_1-protocol.rst

```diff
@@ -128,4 +128,28 @@ The following example defines a service that requires the use of
 .. |protocol test link| replace:: https://github.com/awslabs/smithy/tree/main/smithy-aws-protocol-tests/model/awsJson1_1
 .. include:: aws-json.rst.template
 
+
+--------------------
```

Comment:
I think we should prohibit renames for each AWS protocol, not just aws-json-1.1. That would probably mean you'd want to break this content out into an include thing with a .txt file to not have to copy/paste so much in the docs of each protocol. That or make a section at the top level of the protocol docs to document this and link to it from each protocol doc (e.g., here https://smithy.io/2.0/aws/protocols/index.html)

Reply:
Makes sense. I have simplified the documentation for now and not go into the error shape ID/name details because each protocol behaves differently.